packages/augur-sdk/src/utils.ts

```diff
@@ -165,3 +166,52 @@ export function calculatePayoutNumeratorsArray(
   }
   return payoutNumerators;
 }
+
+export function getOutcomeDescriptionFromOutcome(
+  outcome: number,
+  market: MarketCreatedLog
+): string {
+  if (market.marketType === MarketType.YesNo) {
+    if (outcome === 0) {
+      return 'Invalid';
+    } else if (outcome === 1) {
+      return 'No';
+    } else {
+      return 'Yes';
+    }
+  } else if (market.marketType === MarketType.Scalar) {
+    if (outcome === 0) {
+      return 'Invalid';
+    } else if (outcome === 1) {
+      return new BigNumber(market.prices[0]).toString(10);
+    } else {
+      return new BigNumber(market.prices[1]).toString(10);
+    }
+  } else {
+    return toAscii(market.outcomes[new BigNumber(outcome).toNumber()]);
+  }
+}
+
+export function getOutcomeFromPayoutNumerators(payoutNumerators: BigNumber[]): number {
```

Comment:
I see this has moved, it's not needed there is already a method that converts payout numerators to outcome. Use the existing one.

Reply:
I'm not using this anywhere but I'm not going to refactor the Accounts getters in this PR either so I've left a TODO for that work.

Reply:
if I'm reading this correct `getOutcomeFromPayoutNumerators` doesn't handle converting scalar market payout numerators into outcome price